src/pytorch_lightning/serve/servable_module.py

```diff
@@ -0,0 +1,15 @@
+from modulefinder import Module
+from typing import Any, Callable, Dict, Tuple
+
+import torch
+
+
+class ServableModule(Module):
```

Comment:
Could you add some docs here about what each of these functions stand for? Doesn't have to be elaborate but perhaps with some examples as you have in the PR description?

Reply:
Done !